### PR TITLE
LPS-155635 | Fix Poshi Automation Master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-155635

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithAdvancedConfiguration.testcase [hunk 1]
```diff
@@ -340,9 +340,9 @@ definition {
 		}
 	}
 
-	@description = "This is a test for LPS-150921. The user could view message when the fragment is hidden. The Hide from Site Search Results is not grouped in fieldset on Advanced panel."
+	@description = "This is a test for LPS-150921. The Hide from Site Search Results is not grouped in fieldset on Advanced panel."
 	@priority = "4"
-	test ViewMessageWhenHideFragment {
+	test ViewHideFromSearchNotGroupedInFieldSet {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithAdvancedConfiguration.testcase:345]: ViewHideFromSiteSearchResultsNotGroupedInFieldset

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind modifyng according to comments? Thanks

**Comment by ozysouza** [comment]
ci:forward

**Pull request closed without merging**